tpmutil/tbs/tbs_windows_test.go

```diff
@@ -0,0 +1,54 @@
+package tbs
+
+import (
+	"os"
+	"testing"
+)
+
+func getContext(t *testing.T) Context {
+	ctx, err := CreateContext(TPMVersion20, IncludeTPM12|IncludeTPM20)

```

Comment:
Are all Windows machines guaranteed to support this?
We should maybe not fail the test when running on a device without a TPM, like all the other integration tests

Reply:
`getContext()` now skips instead of failing (this matches what we do for the tpm library's integration tests)